api/client/proto/authservice.proto

```diff
@@ -977,6 +1019,89 @@ message DeleteWindowsDesktopRequest {
     string Name = 1 [ (gogoproto.jsontag) = "name" ];
 }
 
+// ChangePasswordWithTokenRequest defines a request to change one or more of a user's auth
+// credential.
+message ChangePasswordWithTokenRequest {
```

Comment:
Maybe `ChangeAuthenticationRequest` and `rpc ChangeAuthentication(...)`, so it doesn't seem tied to passwords.